x-pack/plugins/stack_connectors/public/connector_types/cases_webhook/webhook_connectors.test.tsx

```diff
@@ -69,27 +71,31 @@ describe('CasesWebhookActionConnectorFields renders', () => {
-    expect(getByTestId('webhookPasswordInput')).toBeInTheDocument();
-    expect(getByTestId('webhookHeadersKeyInput')).toBeInTheDocument();
-    expect(getByTestId('webhookHeadersValueInput')).toBeInTheDocument();
-    expect(getByTestId('webhookCreateMethodSelect')).toBeInTheDocument();
-    expect(getByTestId('webhookCreateUrlText')).toBeInTheDocument();
-    expect(getByTestId('webhookCreateIncidentJson')).toBeInTheDocument();
-    expect(getByTestId('createIncidentResponseKeyText')).toBeInTheDocument();
-    expect(getByTestId('getIncidentUrlInput')).toBeInTheDocument();
-    expect(getByTestId('getIncidentResponseExternalTitleKeyText')).toBeInTheDocument();
-    expect(getByTestId('viewIncidentUrlInput')).toBeInTheDocument();
-    expect(getByTestId('webhookUpdateMethodSelect')).toBeInTheDocument();
-    expect(getByTestId('updateIncidentUrlInput')).toBeInTheDocument();
-    expect(getByTestId('webhookUpdateIncidentJson')).toBeInTheDocument();
-    expect(getByTestId('webhookCreateCommentMethodSelect')).toBeInTheDocument();
-    expect(getByTestId('createCommentUrlInput')).toBeInTheDocument();
-    expect(getByTestId('webhookCreateCommentJson')).toBeInTheDocument();
+    expect(await screen.findByTestId('authNone')).toBeInTheDocument();
+    expect(await screen.findByTestId('authBasic')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookUserInput')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookPasswordInput')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookHeadersKeyInput')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookHeadersValueInput')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookCreateMethodSelect')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookCreateUrlText')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookCreateIncidentJson')).toBeInTheDocument();
+    expect(await screen.findByTestId('createIncidentResponseKeyText')).toBeInTheDocument();
+    expect(await screen.findByTestId('getIncidentUrlInput')).toBeInTheDocument();
+    expect(
+      await screen.findByTestId('getIncidentResponseExternalTitleKeyText')
+    ).toBeInTheDocument();
+    expect(await screen.findByTestId('viewIncidentUrlInput')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookUpdateMethodSelect')).toBeInTheDocument();
+    expect(await screen.findByTestId('updateIncidentUrlInput')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookUpdateIncidentJson')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookCreateCommentMethodSelect')).toBeInTheDocument();
+    expect(await screen.findByTestId('createCommentUrlInput')).toBeInTheDocument();
+    expect(await screen.findByTestId('webhookCreateCommentJson')).toBeInTheDocument();
   });
-  test('Toggles work properly', async () => {
-    const { getByTestId, queryByTestId } = render(
+
+  it('Toggles work properly', async () => {
```

Comment:
nit: Could we change the title to something that describes the new functionality?